lib/Jsrt/Core/JsrtCore.cpp

```diff
@@ -149,10 +149,9 @@ JsSetModuleHostInfo(
     Js::SourceTextModuleRecord* moduleRecord;
     if (!Js::SourceTextModuleRecord::Is(requestModule))
     {
-        if (moduleHostInfo != JsModuleHostInfo_FetchImportedModuleCallback &&
-            moduleHostInfo != JsModuleHostInfo_FetchImportedModuleFromScriptCallback &&
-            moduleHostInfo != JsModuleHostInfo_NotifyModuleReadyCallback &&
-            moduleHostInfo != JsModuleHostInfo_InitializeImportMetaCallback)
+        if (moduleHostInfo == JsModuleHostInfo_Exception ||
```

Comment:
Invert the condition as the number of callback types is now greater than the number of non-callbacks